build/spark-build-info

```diff
@@ -32,7 +32,7 @@ echo_build_properties() {
   echo revision=$(git rev-parse HEAD)
   echo branch=$(git rev-parse --abbrev-ref HEAD)
   echo date=$(date -u +%Y-%m-%dT%H:%M:%SZ)
-  echo url=$(git config --get remote.origin.url)
+  echo url=$(git config --get remote.origin.url |  sed 's|https://\(.*\)@\(.*\)|https://\2|')
```

Comment:
BTW does it cover the case when the password contains `@`?

Reply:
Yes,

```
 echo "https://user:p@ssw@#$rd@git.apache.org/Spark" |  sed 's|https://\(.*\)@\(.*\)|https://\2|'

https://git.apache.org/Spark

```


Secondly, passwords are uri encoded so `@` is converted to `%40`.